lib/ruby_lsp/executor.rb

```diff
@@ -222,6 +222,16 @@ def run(request)
         workspace_symbol(request.dig(:params, :query))
       when "rubyLsp/textDocument/showSyntaxTree"
         show_syntax_tree(uri, request.dig(:params, :range))
+      when "rubyLsp/workspace/dependencies"
+        dep_keys = Bundler.definition.locked_deps.keys.to_set
+        Bundler.definition.specs.map do |spec|
```

Comment:
I think we had a few cases where `definition.to_spec` returns `nil` (e.g. #1326). Can `definition.specs` also contain `nil`?

Reply:
I don't think so, since `definition.specs` should map directly to the list of specs in the `GEM` section of a `Gemfile.lock`.

Btw, the issue you mention is a RubyGems issue, not a Bundler issue, whereas the calls we are making here are purely to Bundler.